multiqc/templates/default/head.html

```diff
@@ -16,8 +16,26 @@
 <title>{{ config.title + ': ' if config.title != None }}MultiQC Report</title>
 
 <!-- JSON plot data -->
+<script type="text/plain" id="mqc_compressed_plotdata">{{ report.plot_compressed_json }}</script>
+
+<script type="application/json" id="mqc_config">{{
+{
+    "num_datasets_plot_limit": config.num_datasets_plot_limit,
+    "sample_names_rename": config.sample_names_rename,
+    "decimalPoint_format": config.decimalPoint_format,
```

Comment:
I think that this can print the text `None` if it's not set, which breaks stuff. This is why it was inside a conditional statement in the old code.

Reply:
This combination works well because ` | tojson` converts None into null

Problematic code:
```html
<script type="application/json" id="mqc_config">
{
    "num_datasets_plot_limit": {{config.num_datasets_plot_limit}},
    "sample_names_rename": {{config.sample_names_rename | tojson}},
    "decimalPoint_format": "{{config.decimalPoint_format}}",
    "thousandsSep_format": "{{config.thousandsSep_format}}",
}
</script>
```

Reply:
Ok great, thanks!